app/views/forever_style_guide/sections/components/_modals.erb

```diff
@@ -0,0 +1,119 @@
+    <div class="modal-content">
+      <div class="modal-header modal-theme color_block-primary">
+        <span type="button" class="modal-close-position" data-dismiss="modal"><i class="fa fa-close fa-2x"></i></span>
+        <h3 class="modal-title"><i class="fa fa-check-circle-o"></i> 1 Item Added to Cart</h3>
+      </div>
+      <div class="modal-body modal-body-info">
+      <div class='row'>
+        <div class='col-md-6'>
+         <%= image_tag("forever_style_guide/single-product.jpg", alt: "Forever Logo", class:"navbar-brand-logo") %>
+        </div>
+        <div class='col-md-6'>
+          <h3>What Makes Your Soul Happy - 5x7 Digital Art Print</h3>
+          <p>Designed by <%= link_to 'LJS Designs', '#', title: 'LJS Designs' %><br />Price $3.50</p>
+          <button type="button" class="btn btn-secondary btn-inverse" data-dismiss="modal">Keep Shopping</button>
+          <button type="button" class="btn btn-primary" data-dismiss="modal">Checkout</button>
+        </div>
+      </div>
+      </div>
+      <div class="modal-footer modal-footer-products">
+        <div class='row text-center'>
+          <p>Shoppers also liked:</p>
+          <div class='col-md-4 product-preview'><%= image_tag("forever_style_guide/digiart1.png", alt: "Forever Logo", class:"navbar-brand-logo") %>
+            <p class='modal-product-name-sm'>Jammin' Jungle Juice MEGA Bundle</p>
+            <%= link_to 'View Details', '#', title: 'View Details' %>
+          </div>
+          <div class='col-md-4 product-preview'><%= image_tag("forever_style_guide/digiart2.png", alt: "Forever Logo", class:"navbar-brand-logo") %>
+            <p class='modal-product-name-sm'>Sunshine on My Shoulder - The Bundle!</p>
+            <%= link_to 'View Details', '#', title: 'View Details' %>
+          </div>
+          <div class='col-md-4 product-preview'><%= image_tag("forever_style_guide/digiart3.png", alt: "Forever Logo", class:"navbar-brand-logo") %>
+            <p class='modal-product-name-sm'>Vintage Americana Bundle</p>
+             <%= link_to 'View Details', '#', title: 'View Details' %>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- Media Modals -->
+<div id="media" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header modal-theme color_block-artisan">
+        <span type="button" class="modal-close-position" data-dismiss="modal"><i class="fa fa-close fa-2x"></i></span>
+        <h3 class="modal-title"><i class="fa fa-info-circle"></i> Why not round 'em up with Lasso Select?</h3>
+      </div>
+      <div class="modal-body modal-body-info">
+        <%= image_tag("forever_style_guide/contextualhelplassoanimation.gif", alt: "Forever Logo", class:"navbar-brand-logo") %>
+        <p class='media text-center'>Select and de-select lots of photos and documents by clicking and dragging your cursor around the items you want to work on.</p>
```

Comment:
this media class is pretty ambiguous. I think modal-media-text or something like that would be more explicit.